source/views/building-hours/lib/__tests__/moment.helper.js

```diff
@@ -3,10 +3,28 @@ import moment from 'moment-timezone'
 export {moment}
 
 const CENTRAL_TZ = 'America/Chicago'
+const baseTime = moment('2019-12-18T18:39:45').tz(CENTRAL_TZ)
+
+export const dayMoment = (time: string, format: ?string = 'ddd h:mma') => {
+	let parsed = moment.tz(time, format, false, CENTRAL_TZ)
+
+	let dow = parsed.day()
+
+	parsed
+		.year(baseTime.year())
+		.month(baseTime.month())
+		.date(baseTime.date())
+		.day(dow)
```

Comment:
Nit: change `dow` -> `dayOfWeek` for legibility?

Reply:
Done.